.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
+
+---
+
+## Before
+
+< The current status >
+
+## After
+
+< What will be changed >
+
+## Screenshots
+
+< If required. [LICEcap](http://www.cockos.com/licecap/) for Mac or SilentCast for Linux allow you to great GIFs from screen capture >
+
+## Comments
```

Comment:
I think the template should include a "Technical Details" section, before this section

## Technical notes
_If the PR introduces noteworthy technical changes, please describe them here. Provide code snippets if necessary_

## Additional comments
_Anything you would like the reviewer to note_